DigitalPuzzle/LedStrip.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+enum WireType {H = 1, L = 0, Z = -1, NC = -2};
+
+class LedStrip {
+public:
+	LedStrip();
+	LedStrip(WireType type, int greenPin, int redPin);
+
+	WireType Type;
```

Comment:
Type moet eigenlijk value heten want heeft helemaal niks te maken met type. Dus:
```suggestion
	WireValue Value;
```